.gitignore

```diff
@@ -48,3 +48,6 @@ cypress/videos
 
 
 **/content/compiled-locales
+
+# saleor-app-sdk - APL for local app development
+.saleor-app-auth.json
```

Comment:
This should be ignored, contain tokens for local development

Reply:
TODO - why graphql.schema.json and graphql.index.ts commited, shouldnt they be in gitignore?

Reply:
I don't think we use graphql.schema.json so it can be removed completely. graphql.index.ts should stay.